### PR TITLE
Feature/infra - deployment of sample project with Pulumi

### PR DESCRIPTION
**Pull request by karpikpl**
Pulumi deployment project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -752,3 +752,6 @@ __azurite_db_blob_extent__.json
 __blobstorage__/
 docker-compose.local.override.yml
 __azurite_db_*
+
+# Pulumi
+Pulumi.dev.yaml
```
Comment (karpikpl) [inline .gitignore:757]: might be easier to provide a stack file?
Describe stack in readme or pass in params when calling pulumi CLI

Comment (karpikpl) [reply .gitignore:757]: done

File: samples/My.Hr/My.Hr.Database/My.Hr.Database.csproj [hunk 2]
```diff
@@ -21,13 +21,14 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="DbEx" Version="1.0.13" />
+    <!-- <PackageReference Include="DbEx" Version="1.0.13" /> -->
```
[the remark this answers is not in the document]
Comment (karpikpl) [reply samples/My.Hr/My.Hr.Database/My.Hr.Database.csproj:24]: done

File: samples/My.Hr/My.Hr.Infra/Components/Apps.cs [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        // set to 1 if app is going to be deployed separately 
+        Output<string> appRunFromPackage = Output.Create("1");
+        Output<string> funcRunFromPackage = Output.Create("1");
+
+        args.IsAppDeploymentEnabled.Apply(isEnabled =>
+        {
+            if (isEnabled)
+            {
+                Log.Info("Setting up deployments from zip for the app and function");
+                appRunFromPackage = PrepareAppForDeployment("app", "../My.Hr.Api/bin/Debug/net6.0/publish");
```
[the remark this answers is not in the document]
Comment (karpikpl) [reply samples/My.Hr/My.Hr.Infra/Components/Apps.cs:38]: done

**Review by chullybun — APPROVED** [review]

**Pull request merged**